pkg/project/operations.go

```diff
@@ -165,33 +153,26 @@ func List(ctx context.Context, opts Options) (projects []string, warnings map[st
 		}
 	}
 
+	managerHost, _, managerHostExists := strings.Cut(cfg.CurrentProject, "/")
+
 	creds, err := credentials.NewLocalOnlyStore(cfg)
 	if err != nil {
 		return nil, nil, err
 	}
 
-	var (
-		wg     sync.WaitGroup
-		result = make(chan listResult)
-	)
+	var result listResult
 	warnings = map[string]error{}
 
-	listLocalKubeconfig(ctx, &wg, result, opts)
-	if !onlyListLocalKubeconfig {
-		listAcornServer(ctx, &wg, creds, cfg, result)
+	if onlyListLocalKubeconfig || !managerHostExists {
+		result = listLocalKubeconfig(ctx, opts)
+	} else {
+		result = listAcornServer(ctx, managerHost, creds)
```

Comment:
This `List` function is also used when the user runs `acorn projects`. Should we still return all projects in that case? If so, I don't think this will work in this case.

Reply:
Oh, that is a good point that I did not realize before. Should I just add another parameter to this function which specifies whether to perform this one-server filtering (and restore the old functionality)?

Reply:
Yeah, I think that make sense to me.